index.yaml

```diff
@@ -2,35 +2,23 @@ apiVersion: v1
 entries:
   ego:
   - apiVersion: v1
-    created: "2020-02-15T22:14:30.745294704-05:00"
-    dependencies:
-    - name: postgres
-      repository: https://overture-stack.github.io/charts-server
-      version: 0.2.0
```

Comment:
@hlminh2000 why did the dependencies disappear

Reply:
hmmm I'm not sure, how was are the dependencies specified?

Reply:
I didn't touch ego though...

Reply:
@henro001 @hlminh2000 i found out why it was deleted! so when i use `helm v2.14.3` to create the index, it deletes the dependencies (happened for score and song), however when i use `helm v3.1.0` the dependencies are maintained. Ensure you are using v3.1.0 to create the index.yaml